openpype/hosts/houdini/plugins/create/create_composite.py

```diff
@@ -35,8 +37,20 @@ def create(self, subset_name, instance_data, pre_create_data):
             "copoutput": filepath
         }
 
+        if self.selected_nodes:
+            if len(self.selected_nodes) > 1:
+                raise CreatorError("More than one item selected.")
```

Comment:
could be a nice feature 😋

Reply:
Haha, something for another time I'd say 🙉 🙈 🙊

Reply:
hahaha totally